activerecord/lib/active_record/migration/command_recorder.rb

```diff
@@ -69,7 +69,7 @@ def record(*command, &block)
       # invert the +command+.
       def inverse_of(command, args, &block)
         method = :"invert_#{command}"
-        raise IrreversibleMigration unless respond_to?(method, true)
+        raise IrreversibleMigration, "#{method} is not defined, so #{command} is not reversible" unless respond_to?(method, true)
```

Comment:
I don't think we should talk about the `method` here. It is something internal to Rails, users don't need to care about it. Maybe something like: "This migration uses #{command} that is not automatically reversible, please ensure you have a `down` method"?